python/tskit/tables.py

```diff
@@ -2795,3 +2797,18 @@ def union(
             self.provenances.add_row(
                 record=json.dumps(provenance.get_provenance_dict(parameters))
             )
+
+    def find_ibd(self, samples, max_time=None, min_length=0):
+        """
+        Docstring for method goes in here.
+        """
```

Comment:
```python
def find_ibd(self, samples, max_time=None, min_length=None):
     max_time = sys.float_info.max if max_time is None else max_time
     min_length = 0  if min_length is None else min_length
     return self._ll_tables.find_ibd(samples, max_time=max_time, min_length=min_length)
```
It's better to avoid hard-coding the default values of parameters into the signature unless we're very sure they'll never change.